src/Compilers/CSharp/Test/Semantic/Diagnostics/DiagnosticAnalyzerTests.cs

```diff
@@ -2913,5 +2913,69 @@ public void TestSymbolStartAnalyzer_NestedOperationAction_Inside_AllSymbolKinds(
                 Diagnostic("SymbolStartRuleId").WithArguments("e1", "Analyzer5").WithLocation(1, 1),
                 Diagnostic("SymbolStartRuleId").WithArguments("f1", "Analyzer6").WithLocation(1, 1));
         }
+
+        [Fact, WorkItem(32702, "https://github.com/dotnet/roslyn/issues/32702")]
+        public void TestInvocationInPartialMethod()
```

Comment:
>TestInvocationInPartialMethod [](start = 20, length = 29)

Does it make sense to add direct tests for the component that we are fixing? Imagine that the fix is undone by mistake, would these tests appear flaky because they will be passing one time and fail another?

Reply:
I did run the added tests bunch of times prior to the fix and they failed consistently. However, I agree with you that we probably need a better deterministic testing strategy for symbol start/end action based analyzers - I'd like to delay that work until we do see flakiness in current tests though.